datafeeder/src/main/java/org/georchestra/datafeeder/service/publish/impl/GeorchestraPublishingServicesConfiguration.java

```diff
@@ -32,10 +38,33 @@
  * @see DataFeederConfigurationProperties
  */
 @Configuration
-@Profile({ "!mock" })
+@Profile("!mock")
 public class GeorchestraPublishingServicesConfiguration {
 
     public @Bean DataBackendService dataBackendService() {
         return new GeorchestraDataBackendService();
     }
+
+    public @Bean OWSPublicationService owsPublicationService() {
+        return new GeorchestraOwsPublicationService();
+    }
+
+    public @Bean GeoServerClient geoServerApiClient(DataFeederConfigurationProperties props) {
+        ExternalApiConfiguration config = props.getPublishing().getGeoserver();
+        String restApiEntryPoint = config.getApiUrl().toExternalForm();
+
+        GeoServerClient client = new GeoServerClient(restApiEntryPoint);
+
+        Map<String, String> authHeaders = new HashMap<>();
+        // authHeaders.put("sec-proxy", "true");
```

Comment:
sec-proxy is not mandatory for this use case indeed

Reply:
Right, superseded by https://jira.camptocamp.com/browse/GSGEODPT43-139